magicblock-accounts/src/external_accounts_manager.rs

```diff
@@ -378,7 +378,7 @@ where
         // Therefore we use the current slot which could result in two commits with the same
         // slot. However since we most likely will phase out frequent commits we accept this
         // inconsistency for now.
-        static MESSAGE_ID: AtomicU64 = AtomicU64::new(u64::MAX - 1);
+        static MESSAGE_ID: AtomicU64 = AtomicU64::new((i64::MAX - 1) as u64);
```

Comment:
I was guessing that we must be using `u64` as ID. So this change makes total sense. 👍

Reply:
I'm not fully understanding why we have to adapt this here.
Why not convert it right when writing to Sqlite and reading it like we do for other u64s?

Reply:
That also works, actually would better align with how other values are treated

Reply:
Done